### PR TITLE
Updates version of ember-a11y-refocus

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

If merged, this PR updates the version of `ember-a11y-refocus`.  It's a patch version that resolves issues TFC was having with the new `excludeAllQueryParams` flag. 

Tested this with @deanmarano (who can confirm) and this resolves the issue he was seeing in Terraform.

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

**Comment by Dhaulagiri** [comment]
You'll want to add a `patch` changeset for the change to the components package

**Review by zamoore — APPROVED** [review]
Do we need a changeset for this?

**Comment by MelSumner** [comment]
@Dhaulagiri any idea why adding a changeset would cause Percy to fail? 🤔

**Comment by Dhaulagiri** [comment]
Not 100% sure but likely something got merged to main that isn't in your branch. I suspect rebasing this branch on main would clear that up.

File: .changeset/poor-rules-flow.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Update version of ember-a11y-refocus
```
Comment (Dhaulagiri) [inline .changeset/poor-rules-flow.md:5]: In the interest of explicitness in our changelog
```suggestion
Update  `ember-a11y-refocus` to `4.1.3`
```

**Comment by MelSumner** [comment]
Something in my git rebase didn't work correctly, so I'm just going to redo this PR.

**Pull request closed without merging**